java/geodesic-operations/README.md

```diff
@@ -0,0 +1,33 @@
+# Geodesic Operations 
+Demonstrates how to use the GeometryEngine to calculate a geodesic path between two points and measure its distance.
+
+![Geodesic Operations Image](GeodesicOperations.png)
```

Comment:
There's a script which scrapes out these files for various things like the webpage and the sample viewer. In order for it to work the screenshots must be of the format: geodesic-operations.png

Don't forget to rename the screenshot too :)

Reply:
For standardization purposes, call it 'Geodesic Operations App' instead of 'Geodesic Operations Image'